python/helper-image/launcher/launcher.go

```diff
@@ -64,10 +68,24 @@ limitations under the License.
 // python -m pydevd --server --port 5678 --DEBUG --continue \
 //   --file /tmp/pydevd716531212/skaffold_pydevd_launch.py
 // ```
+//
+// The launcher can be configured through several environment
+// variables:
```

Comment:
Is there a recommended way for skaffold users to configure the env vars of the helper image?  If so perhaps it would make sense add a blurb/doc-somwhere on how a skaffold user should set these.

Reply:
Hoping that the users never need to do this.  But yes, it would be useful to document at least `WRAPPER_ENABLE`.